ofl/andika/METADATA.pb

```diff
@@ -1,5 +1,5 @@
 name: "Andika"
-designer: "SIL International"
+designer: "SIL International, Victor Gaultney, Annie Olsen, Julie Remington, Don Collingsworth, Eric Hays, Becca Hirsbrunner"
```

Comment:
@jvgaultney can you confirm this is the correct credit for the fonts? Other possibility would be to have SIL would as the only credit, and all the individual authors mentioned in the description.